physionet-django/console/templates/console/complete_application_display.html

```diff
@@ -1,82 +1,87 @@
-  <div class="card-header">
-    <h2 style='font-size: 1.2rem;' id='application_{{ application.id }}'>{{ application.get_full_name }}</h2>
+  <div class="card-header" id="application_{{ application.id }}">
+    <h2 style="font-size: 1.2rem;">{{ application.get_full_name }}</h2>
     <p>
-      <a class="" href='{{ application.mailto }}'>
+      <a href='{{ application.mailto }}'>
 	Send e-mail to applicant
       </a><br>
       Database requested: {{ application.project_of_interest }}
     </p>
     <form action="" method="post" class="form-signin"> <!-- empty action generates error! -->
       <p>
-		{% csrf_token %}
-		<button class="" name="contact_reference" value="{{ application.id }}" type="submit" style="margin-right: 1em;">
-		  Contact Reference
-		</button> 
-		{{ application.reference_email }} 
-		{% if application.reference_contact_datetime %}
-			[The reference was contacted on {{ application.reference_contact_datetime }}]
-		{% endif %}
-	    {% if application.known_ref %} [Reference is known] {% endif %}
+	{% csrf_token %}
+	<button name="contact_reference" value="{{ application.id }}" type="submit" style="margin-right: 1em;">
+	  Contact Reference
+	</button> 
+	{{ application.reference_email }} 
+	{% if application.reference_contact_datetime %}
+		[The reference was contacted on {{ application.reference_contact_datetime }}]
+	{% else %}
+        {% if application.known_ref %}
+        	[Reference is known]
+        {% endif %}
+	{% endif %}
       </p>
     </form>
-    <form action="" method="post" class="form-signin">  <!-- empty action generates error! -->
+    <form method="post" class="form-signin">  <!-- empty action generates error! -->
       <p>
-		{% csrf_token %}
-	      <div class="form-group row">
-		  	<label for="inputPassword">Reason for approval:</label>
-		  	<div class="col-sm-6">
-		    	{{ process_credential_form.responder_comments }}
-		  	</div> <!-- col-sm-6 -->
-		  	<div class="col-sm-4">
-		    	<button class="" type="submit" name="status" value="{{ process_credential_form.status.field.choices.2.0 }}">
-	              {{ process_credential_form.status.field.choices.2.1 }}
-		    	</button>
-		  	</div> <!-- col-sm-4 -->
-	      </div> <!-- form-group row -->
-	      <input type='hidden' name='process_application' value="{{ application.id }}">
-		  {% for error in process_credential_form.responder_comments.errors %}
-			<div class="alert alert-danger">
-		  	  <strong>{{ error|escape }}</strong>
-			</div> <!-- alert alert-danger -->
-		  {% endfor %}
-		  {% for error in process_credential_form.non_field_errors %}
-			<div class="alert alert-danger">
-		  	  <strong>{{ error|escape }}</strong>
-			</div> <!-- alert alert-danger -->
-		  {% endfor %}
+	{% csrf_token %}
+        <div class="form-group row">
+	  <label for="inputPassword">Reason for approval:</label>
+	  <div class="col-sm-6">
+	    {{ process_credential_form.responder_comments }}
+	  </div> <!-- col-sm-6 -->
```

Comment:
What is the purpose of `<!-- col-sm-6 -->`  comment? Can this be removed?

Reply:
Its good to know when the divs end, at least for this code...

Reply:
yep, makes sense.